src/meta/attribute.rs

```diff
@@ -1062,23 +1090,159 @@ impl ChannelList {
     }
 }
 
+// https://github.com/AcademySoftwareFoundation/openexr/blob/master/src/lib/OpenEXR/ImfTimeCode.cpp
 impl TimeCode {
 
     /// Number of bytes this would consume in an exr file.
     pub const BYTE_SIZE: usize = 2 * u32::BYTE_SIZE;
 
-    /// Without validation, write this instance to the byte stream.
+    /// Returns an error if this time code is considered invalid.
+    pub fn validate(&self, strict: bool) -> UnitResult {
+        if strict {
+            if self.frame > 29 { Err(Error::invalid("time code frame larger than 29")) }
```

Comment:
> `Error::invalid("time code frame larger than 29")`

I'm not sure 29 is the limit: https://github.com/AcademySoftwareFoundation/openexr/blob/master/src/lib/OpenEXR/ImfTimeCode.cpp#L212

Reply:
I have a problem with this: The value has 6 bits, which allows for 64 values. But the value is converted to a little bit more inefficient data format, "bcd" (binary encoded decimal), which would require more than 6 bits if the range is 0-59. I can't find where the bug is, could you have a quick closer look at this?

Reply:
In timecode.h they said 29 is the limit, and other websites say that too